src/UnitTests/NullBehavior.cs

```diff
@@ -50,27 +50,6 @@ public void Should_map_to_non_null()
         }
     }
 
-    public class When_mapping_untyped_null_to_IEnumerable : AutoMapperSpecBase
```

Comment:
No mapper would apply here, so I think this should throw after all. Casting the source should work as expected.